packages/components/managed-data-table/src/data-table-settings/data-table-settings.js

```diff
@@ -0,0 +1,155 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import requiredIf from 'react-required-if';
+import SelectInput from '@commercetools-uikit/select-input';
+import Spacings from '@commercetools-uikit/spacings';
+import Constraints from '@commercetools-uikit/constraints';
+import DensityManager from './density-manager';
+import ColumnManager from './column-manager';
+import TABLE_SETTINGS_ACTIONS from '../constants';
+
+const COLUMN_MANAGER = 'columnManager';
+const DISPLAY_SETTINGS = 'displaySettings';
+
+const DataTableSettings = (props) => {
+  const [isColumnManagerOpen, setIsColumnManagerOpen] = useState(false);
+  const [isDisplayManagerOpen, setIsDisplayManagerOpen] = useState(false);
+
+  const dropdownOptions = [
+    ...(props.columnManager.useColumnManager
+      ? [{ value: COLUMN_MANAGER, label: 'Column manager' }]
+      : []),
+    ...(props.displaySettings.useDisplaySettings
+      ? [{ value: DISPLAY_SETTINGS, label: 'Display settings' }]
+      : []),
+  ];
+
+  const handleDropdownChange = (event) => {
+    if (event.target.value === COLUMN_MANAGER) {
+      setIsDisplayManagerOpen(false);
+      setIsColumnManagerOpen(true);
+      return;
+    }
+    setIsColumnManagerOpen(false);
+    setIsDisplayManagerOpen(true);
+  };
+
+  const mappedColumns = props.columnManager.hideableColumns.reduce(
+    (columns, column) => ({
+      ...columns,
+      [column.key]: column,
+    }),
+    {}
+  );
+
+  const selectedColumns = props.columnManager.visibleColumnKeys.map(
+    (columnKey) => mappedColumns[columnKey]
+  );
+
+  return (
+    <Spacings.Stack>
+      <Spacings.Inline justifyContent="space-between" alignItems="center">
+        {props.topBar || <div />}
+        {dropdownOptions.length > 0 && (
+          <Constraints.Horizontal constraint="s">
+            <SelectInput
+              name="form-field-name"
+              value={''}
+              placeholder="Table settings"
```

Comment:
Shouldn't this be translated?

Reply:
The input is never empty. Makes no sense to even have a placeholder